pkg/remediator/completedpoddeleter.go

```diff
@@ -0,0 +1,39 @@
+package remediator
+
+import (
+	"context"
+	"go.uber.org/zap"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sync"
+	"time"
+)
+
+type CompletedPodDeleter struct {
+	Base
+}
+
+func (p *CompletedPodDeleter) Run(ctx context.Context, wg *sync.WaitGroup) {
+	defer wg.Done()
+
+	p.reconcileEvery(ctx, p.deleteCompletedPods, 1*time.Hour)
+}
+
+func (p *CompletedPodDeleter) deleteCompletedPods() {
+	p.logger.Info("Running")
+
+	// get completed pods
+	pods, err := p.client.GetPods("", metav1.ListOptions{FieldSelector: "status.phase=Completed"})
```

Comment:
It seems doesn't work, since no `Completed` pod phase: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase